docs/en/advanced-concepts/parallel_tests.md

```diff
@@ -1,32 +1,31 @@
 
-The important flags for automating multiple Android sessions are:
+The important capabilities:
 
-- `-p` the main Appium port
-- `-U` the device id
-- `-bp` the Appium bootstrap port
-- `--chromedriver-port` the chromedriver port (if using webviews or chrome)
-- `--selendroid-port` the selendroid port (if using selendroid)
+- `udid` the device id
+- `chromeDriverPort` the chromedriver port (if using webviews or chrome)
+- `systemPort` If you are using [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver), set a different system port for each Appium instanceset with `systemPort` capability since sometimes there can be a port conflict if different ports aren't used, such as in [this issue](https://github.com/appium/appium/issues/7745).
 
-More information on these flags can be found [here](/docs/en/writing-running-appium/caps.md).
 
-If we had two devices with the ID's 43364 and 32456, we would start two different Appium servers with the following commands:
-
-`node . -p 4492 -bp 2251  -U 32456`
-
-`node . -p 4491  -bp 2252 -U 43364`
-
-As long as your Appium and Appium bootstrap ports are between 0 and 65536, all they have to be is different so that two Appium servers aren't trying to listen on the same port. Be sure that your -u flag corresponds with the correct device ID. This is how Appium knows which device to communicate with, so it must be accurate.
+### Parallel iOS Tests
 
-If you are using chromedriver or selendroid, set a different port for each server.
+With Xcode9 Appium supports parallel RealDevice and Simulator testing. Start your Appium Server on any available port.
 
-If you are using [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver), set a different system port for each Appium instanceset with `systemPort` capability since sometimes there can be a port conflict if different ports aren't used, such as in [this issue](https://github.com/appium/appium/issues/7745).
+The important capabilities:
 
-### Parallel iOS Tests
+#### RealDevice
+- `udid` the device id
+- `wdaLocalPort` unique wdaPort, as WDA defaults to 8100
+ 
+#### Simulator
 
-Unfortunately, running local parallel iOS tests isn't currently possible. Unlike Android, only one version of the iOS simulator can be launched at a time, making it run multiple tests at once.
+- `udid` simulator UDID,this can be retrieved from xcrun simctl list.
+- `deviceName` Simulator Name
+- `platformVersion` Simulator OS version
+- `wdaLocalPort` unique wdaPort, as WDA defaults to 8100
 
-If you do want to run parallel iOS tests, you need to use Sauce. Simply upload your Appium test to Sauce, and it can run as many parallel iOS or Android tests as your account allows. See more about running your tests on Sauce [here](https://docs.saucelabs.com/tutorials/appium/).
+Parallel Safari/Webview sessions are not working due to an Apple bug
```

Comment:
They don't work on Simulator, but real device should be OK by providing _webkitDebugProxyPort_ capability.

Reply:
Done